Buildings/Examples/VAVReheat/Validation/BaseClasses/Floor.mo

```diff
@@ -1,4 +1,4 @@
-﻿within Buildings.Examples.VAVReheat.Validation.BaseClasses;
+within Buildings.Examples.VAVReheat.Validation.BaseClasses;
```

Comment:
This class should not be modified.

Reply:
Actually, it changed because the `master` branch file is encoded with BOM, so the formatting test fails. Not sure how this slipped through the cracks.